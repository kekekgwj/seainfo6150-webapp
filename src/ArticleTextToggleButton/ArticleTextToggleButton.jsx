import React from "react";
import style from "./ArticleTextToggleButton.module.css";
const ArticleTextToggleButton = (props)=> {
    const {
        buttonText,
        onClick
    } = props;
    return (
        <div className={style.ToggleButtonWrapper}>
            <button className={style.ToggleButton} onClick={onClick}>{buttonText}</button>
        </div>
    )
}

export default ArticleTextToggleButton;