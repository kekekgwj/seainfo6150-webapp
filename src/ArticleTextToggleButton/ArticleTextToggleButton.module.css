.ToggleButton {
    height: 30px;
    color: white;
    background-color: red;;
    border-radius: 5px;
    width: 100%;
}
.ToggleButtonWrapper {
    width: 100%;

}