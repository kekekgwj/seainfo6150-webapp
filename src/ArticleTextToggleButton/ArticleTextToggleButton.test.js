import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";

describe("ArticleTextToggleButton tests", () => {
    it("renders correctly", () => {
        const { container } = render(<ArticleTextToggleButton/>)
        expect(container).toMatchSnapshot();
    })
})