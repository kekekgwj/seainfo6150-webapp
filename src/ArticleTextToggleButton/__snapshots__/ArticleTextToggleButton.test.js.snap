// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTextToggleButton tests renders correctly 1`] = `
<div>
  <div
    class="ToggleButtonWrapper"
  >
    <button
      class="ToggleButton"
    />
  </div>
</div>
`;
